includes/class-wc-gateway-stripe-addons.php

```diff
@@ -97,6 +97,26 @@ protected function save_source( $order, $source ) {
 	}
 
 	/**
+	 * Manipulates the payment request specific to subscriptions.
+	 *
+	 * @since 3.1.7
+	 * @version 3.1.7
+	 * @param object $order
+	 * @param object $source
+	 */
+	public function filter_subscription_payment_request( $order, $source ) {
```

Comment:
Sorry that was left there in error. Should have removed that before committing. It is now removed.